cmd/fts/cmd/resume.go

```diff
@@ -0,0 +1,107 @@
+package cmd
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	resumeCmd.Flags().BoolVarP(&dryRun, "dry-run", "d", false, "Run through steps without pushing data to Powergate API.")
```

Comment:
Would be cool to not have to repeat all this configuration for `run` and `resume`. You could make all these persistent flags at the root level, but then the `version` command would inherit them. Another idea is to make "resume" a flag of `run`, and just detect the value of that flag and fork the code path that gets run at that point.

Reply:
yeah, nice. resume as a flag seems like a good call. i could get rid of `run` at that point and make it all work of root

Reply:
Yea, there are so few commands, that could work.

Reply:
I think keeping the `run` command makes sense, and add the `--resume` flag to it. There will be a time when you want to add another top level command next to `run`, and that will be weird if run is just a flag on the root command.